core/src/test/scala/unit/kafka/server/ClientQuotasRequestTest.scala

```diff
@@ -165,29 +163,6 @@ class ClientQuotasRequestTest(cluster: ClusterInstance) {
     ))
   }
 
-  @ClusterTest(types = Array(Type.ZK)) // No SCRAM for Raft yet
-  def testClientQuotasForScramUsers(): Unit = {
```

Comment:
We should make sure we have ported this test to KRaft. Does a KRaft version of this test exist anywhere?

Reply:
There are various SCRAM tests for KRaft. But I agree, let's comment `ClientQuotasRequestTest.testClientQuotasForScramUsers` out rather than removing it, and create a JIRA for converting it to KRaft.

Reply:
We could also change it to KRaft and mark it as `@Disabled` with a JIRA ticket in the description

Reply:
Thanks for the suggestion. Create https://issues.apache.org/jira/browse/KAFKA-17630 for it.